specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -152,6 +152,9 @@
               "$ref": "#/definitions/UsageDetailsListResult"
             }
           },
+          "503": {
```

Comment:
I don't think 503 need to be added separately. It should be same as `default` below.

If you add 503 here, it actually means 503 is now the expected return code, and client should be good with it and proceed.